packages/plugin-json/src/lib/validation.ts

```diff
@@ -79,6 +79,22 @@ export const getDeliveryKeyNotNullableStringReport = (
 const isDeliveryKeyField = (field: FieldDefinitionNode) =>
   hasDirective(field, "amplienceDeliveryKey");
 
+const isMissingExtensionName = (field: FieldDefinitionNode) =>
+  hasDirective(field, "amplienceExtension") &&
+  !field.directives?.some(
+    (d) =>
+      d.name.value === "amplienceExtension" &&
+      d.arguments?.some((arg) => arg.name.value === "name"),
+  );
+
+export const getMissingExtensionNameReport = (
```

Comment:
So I think both of these functions are not needed if we make name a required field